### PR TITLE
Project rework

### PR DESCRIPTION
**Pull request by vshah23**
## Context
[PLG-1252]
We want to set up this getting started app to show users the capabilities of our platform.

## Changes
- Regenerated the project with the Flutter version which is preinstalled on stacks
- Updated the bitrise.yml with a pipeline setup
  - Included exporting apps 
    - Unsigned .app for iOS simulator
    - Unsigned .apk for Android emulator

## Decisions
* Regenerated the project instead of installing the required version of Flutter which would make the build take longer.

[PLG-1252]: https://bitrise.atlassian.net/browse/PLG-1252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.1.2'
+        classpath 'com.android.tools.build:gradle:4.1.0'
```
Comment (ofalvai) [inline android/build.gradle:9]: Is this change intentional? 🫣

File: bitrise.yml [hunk 2]
```diff
@@ -1,64 +1,80 @@
...
-      Next steps:
-      - Check out [Getting started with Flutter apps](https://devcenter.bitrise.io/en/getting-started/getting-started-with-flutter-apps.html).
+workflows:
+  _setup:
     steps:
-    - git-clone@6: { }
-    - flutter-installer@0: { }
-    - cache-pull@2: { }
-    - flutter-test@1:
+    - git-clone: { }
```
Comment (ofalvai) [inline bitrise.yml:30]: We should pin at least the major version of steps

File: bitrise.yml [hunk 3]
```diff
@@ -1,64 +1,80 @@
...
         inputs:
-        - project_location: $BITRISE_FLUTTER_PROJECT_LOCATION
-    - cache-push@2: { }
-    - deploy-to-bitrise-io@2: { }
-
-  deploy:
+        - is_update: false
+  _teardown:
+    steps:
+    - save-dart-cache: { }
```
Comment (ofalvai) [inline bitrise.yml:37]: I'm not sure it's a good idea to save the cache in every stage's every workflow. For example, if I make a mistake that passes the analyze and test workflows, but fails building the iOS/Android app, the cache probably shouldn't be saved.

File: android/build.gradle (hunk 1 above)
Comment (vshah23) [reply android/build.gradle:9]: Yes, I downgraded the project to the version of Flutter which is preinstalled on the stacks in order to speed up the build.

File: bitrise.yml (hunk 2 above)
Comment (vshah23) [reply bitrise.yml:30]: Will take care of this in a follow up PR.

File: bitrise.yml (hunk 3 above)
Comment (vshah23) [reply bitrise.yml:37]: Moved this to the Android export.

File: bitrise.yml (hunk 2 above)
Comment (vshah23) [reply bitrise.yml:30]: https://github.com/bitrise-io/getting-started-flutter-sample-app/pull/12/files
